OpenOversight/app/utils/general.py

```diff
@@ -101,6 +100,9 @@ def normalize_gender(input_gender: str) -> Union[str, None]:
 
     return normalized_genders.get(input_gender.lower().strip())
 
+def str_to_bool(val: str) -> int:
+    return val.lower() in ("y", "yes", "t", "true", "on", "1")
```

Comment:
distutils was deprecated in Python 3.10 and removed in Python 3.12. Since we're using Python 3.12 for testing, distutils.util.strtobool had to be replaced.